rllib/core/learner/tests/test_learner.py

```diff
@@ -155,20 +131,26 @@ def test_postprocess_gradients(self):
                 grad_clip=1.0,
                 grad_clip_by="norm",
             )
-            # TODO (sven): remove this once validation does NOT cause HPs to be
-            #  generated anymore.
-            config.validate()
+            # Freeze to be able to get the LearnerGroup config.
             config.freeze()
             learner_group = (
                 config.get_learner_group_config(module_spec=module_spec)
                 .learner(learner_class=config.get_default_learner_class())
                 .build()
             )
             learner = learner_group._learner
+            # Pretend our computed gradients are our weights + 1.0.
+            grads = {
+                learner.get_param_ref(v): v + 1.0
```

Comment:
We should probably just hard code the grads to 1.0 and -1.0 with prob of 50% each (and do not add value to it). If we do this we would be sure that all grad values are 1 and when you clip by value they would become 0.75 or -0.75

Reply:
I'll leave it for the reason of simplicity. If we want to hard-set grads to -1 or 1, we would have to do some shape-magic inside the for-loop (numpy, tf, torch?), whereas right now, the variable `v` takes care of that (framework-agnostic!).

Reply:
got it